components/NewsList.js

```diff
@@ -4,12 +4,7 @@ import styled from 'styled-components';
 export default function NewsList({ title, description, url, urlToImage }) {
   return (
     <span>
-      <ImagetoLink
-        width={240}
-        height={150}
-        src={urlToImage}
-        alt="The image could not be loaded"
-      />
+      <ImagetoLink width={240} height={150} src={urlToImage} alt="urlToImage" />
```

Comment:
Do you have an image description in your data for the alt attribute?

Reply:
desription image to the url. Is this Ok? because i have different description images